### ISSUE TITLE
Triage process for Keras - Tensorflow issues

### ISSUE THREAD

**Issue by Harshini-Gadige** [keras-team/keras#11494]
Hi Gabriel,

I'm part of _Tensorflow Dev Support team_ and we started triaging Tensorflow related issues in the Keras Github repository. As part of this, we are going to follow below steps(drafted on a high level) using the appropriate labels. Hence we are going to create **new** labels(mentioned below) as per our regular standards we follow for other Tensorflow repositories (like Core, Models, TFX etc).

Creating this message to keep you posted on this and please take a look. Feel free to add your thoughts on this. Thank you !

**Steps we follow for triaging :** 

1) Pick an issue related to TF backend and add label type:tensorflow
2) If it is a question for Stack Overflow, add the label "type:support", close the issue and redirect the user to stack overflow.
3) If the user doesn't provide a code snippet to reproduce the error, ask him/her to provide the same.
4) If the issue persists with tf.keras, apply the label "stat:cross-posting to TF" and post it in the Tensorflow core repo.(Once it's triaged there, both the issues in TF core and keras repo can be closed)
5) If the issue doesn't persist with tf.keras, it is assigned to the Keras person based on the Round Robin rotation.
6) Once the issue is fixed, it can be closed.

**New labels we are going to create :** 

stat:awaiting response  -> When triage/Dev person asks some information and waiting for the user to respond
stat:awaiting tensorflower -> When assigned to a Developer and waiting for his/her response
stat:contributions welcome -> When a feature is open for external contributions 
stat:cross-posting to TF -> when the issue persists with tf.keras
type:bug/performance -> When it is a bug or any performance related issue
type:docs -> Issue related to any changes to be made in the documents or in the examples page of the site
type:support -> When a question is asked by user or when user seeks consultation(no bug/feature request) then we close the issue asked the user to post in Stack Overflow and mark it with this label.
type:build/install -> anything related to build or installation
type:feature -> If a user asks for a new feature 
type:tensorflow -> For all TF related issues.

@dksb @fchollet @karmel   -  Hi, feel free to add your inputs.

**Comment by gabrieldemarmiesse** [comment]
First of all, thank you for the support :)

Since I'm already putting tags on issues, here are the labels that I'll put:
`type:tensorflow`
`type:support` (previously `question`)
`stat:contributions welcome` (previously `help wanted`)
`type:bug`
`type:performance`
`type:docs` (previously `documentation`)
`type:install/build`
`type:feature` (previously `feature request`)
`announcement`
`to investigate` (when we don't know if it's a bug or a user error, which happens often. Ideally should be converted into `user error` or `type:bug` or `type:performance`)
`good first issue` (very useful for newcomers so that they can get into the codebase progressively, this tag proved itself very effective recently)

Also for consistency, I'll use the same labels for tensorflow, theano and cntk issues. The only difference will be the tag `type: tensorflow`.

One question that I have, do we really have to use `type: tensorflow` or `stat: contribution welcome`? Isn't `Tensorflow` or `Contribution welcome` explicit enough? What's the added value of those prefixes? Is it for bots?

**Comment by dksb** [comment]
@gabrieldemarmiesse  Hi Gabriel, as you know GitHub doesn't do a great job on organizing issues. We have categories of labels that we prefix with category type (like type:, stat:). Since we support more than one repo, prefix helps us easily know what labels belong to a type of category for that repo. It helps standardize the metrics reporting and the bot based tooling.

**Comment by dksb** [comment]
Yes, that sounds good to us. We will pickup issues that are labelled type:tensorflow and triage them further. Also, we can mutually revisit this approach if it doesn't work as well as we want.

**Comment by Dref360** [comment]
Hello,
First of all thanks for your support! We managed to lower the issue count to 500 last year or so, but it came back >1500 during the summer. 

Since most issues on Keras are actually questions and we don't strictly force PR to close the related issues. I think we can lower the issue count quite easily. 

Just one quick note: don't forget to redirect users to our Slack/google-group/Stackoverflow. The Slack is quite active and most people can find help there.


## My saved response for redirect
Hi!
This issue isn't related to a bug/enhancement/feature request or other accepted types of issue.

To ask questions, please see the following resources : 
* https://keras.io/
* https://keras.io/#support
* https://gitter.im/Keras-io/Lobby

Thanks!

If you think I made a mistake, please open another issue.


## Saved response for minimal exemple
Hi,
We could work on this better if you could provide a standalone reproducible example.
Some advices:
* Don't use custom data or custom paths.
* Use random arrays or even np.ones, np.zeros.
* The example should run with Keras (and deps) alone.
* Should be Python3 compatible.
* Should not be OS specific.
* The file should reproduce the bug with **high* fidelity.
* Link to a gist would be appreciated.

Thanks!
Dref360

**Comment by Dref360** [comment]
Also, would this help if we shared with your team ours "areas of interest" so that you can assign keras-team's members issues?

**Comment by gabrieldemarmiesse** [comment]
For the minimal example, I suggest this:

* Make a network as small as possible while keeping the bug present.


-------------

> Also, would this help if we shared with your team ours "areas of interest" so that you can assign keras-team's members issues?

I do agree. Round robin is suboptimal since we surely have preferences.

**Comment by harshini-gadige** [comment]
@gabrieldemarmiesse  - Thanks for adding the labels. As we are following specific color code for all the labels, we will change the font and color of the labels accordingly.
